File: assets/downloads/tutorials/extended-game-java/MainMenuScreen.java
package com.badlogic.drop;

import com.badlogic.gdx.Gdx;
import com.badlogic.gdx.Screen;
import com.badlogic.gdx.graphics.Color;
import com.badlogic.gdx.utils.ScreenUtils;

public class MainMenuScreen implements Screen {

	final Drop game;

	public MainMenuScreen(final Drop game) {
		this.game = game;
	}

	@Override
	public void render(float delta) {
		ScreenUtils.clear(Color.BLACK);

		game.viewport.apply();
		game.batch.setProjectionMatrix(game.viewport.getCamera().combined);

		game.batch.begin();
		//draw text. Remember that x and y are in meters
		game.font.draw(game.batch, "Welcome to Drop!!! ", 1, 1.5f);
		game.font.draw(game.batch, "Tap anywhere to begin!", 1, 1);
		game.batch.end();

		if (Gdx.input.isTouched()) {
			game.setScreen(new GameScreen(game));
			dispose();
		}
	}

	@Override
	public void resize(int width, int height) {
		game.viewport.update(width, height, true);
	}

	@Override
	public void show() {
	}

	@Override
	public void hide() {
	}

	@Override
	public void pause() {
	}

	@Override
	public void resume() {
	}

	@Override
	public void dispose() {
	}
}

File: assets/downloads/tutorials/extended-game-kotlin/MainMenuScreen.kt
package drop

import com.badlogic.gdx.Gdx
import com.badlogic.gdx.Screen
import com.badlogic.gdx.graphics.OrthographicCamera
import com.badlogic.gdx.utils.ScreenUtils

class MainMenuScreen(private val game: Drop): Screen {

    private var camera = OrthographicCamera().apply {
        setToOrtho(false, 800f, 480f)
    }

    override fun render(delta: Float) {
        ScreenUtils.clear(0f, 0f, 0.2f, 1f);

        camera.update();
        with (game.batch) {
            projectionMatrix = camera.combined;

            begin();
            game.font.draw(this, "Welcome to Drop!!! ", 100f, 150f);
            game.font.draw(this, "Tap anywhere to begin!", 100f, 100f);
            end();
        }

        if (Gdx.input.isTouched) {
            game.screen = GameScreen(game);
            dispose();
        }
    }

    override fun show() {

    }

    override fun resize(width: Int, height: Int) {
        
    }

    override fun pause() {
        
    }

    override fun resume() {
        
    }

    override fun hide() {
        
    }

    override fun dispose() {
        
    }
}
